src/design-system/components/badge/Badge.tsx

```diff
@@ -0,0 +1,42 @@
+import { Badge as MuiBadge, BadgeProps as MuiBadgeProps } from '@mui/material'
+import { FC } from 'react'
+
+type BadgeColor = 'primary' | 'success' | 'error'
+
+type SmallBadgeProps = {
+  variant: 'sm'
+  color?: BadgeColor
+  isVisible?: boolean
+}
+
+type LargeBadgeProps = {
+  variant: 'lg'
+  badgeContent: number
+  maxContent?: number
+  color?: BadgeColor
+  isVisible?: boolean
+  showZero?: boolean
```

Comment:
Name it as a boolean, please
```suggestion
  isZeroShown?: boolean
```